### ISSUE TITLE
Segmentation fault when archiving

### ISSUE THREAD

**Issue by priteshshah1983** [ReactKit/SwiftTask#84]
I am able to build and run just fine on simulator and device, but get a segmentation fault only when I try to archive for App Store submission on this line:

```
@discardableResult public func progress(_ progressClosure: @escaping (ProgressTuple) -> Void) -> Self
{
    var dummyCanceller: Canceller? = nil
    return self.progress(&dummyCanceller, progressClosure)
}
```

This is the exact error:
```
@_TTSf4g_n___TTSg5CSo8Progress_T__Ps5Error____TFC9Concierge4Task8progressfFT11oldProgressGSqx_11newProgressx_T_DGS0_xq_q0__ for 'progress' at /Users/pritesh/sampleApp/Vendor/SwiftTask/SwiftTask/SwiftTask.swift:359:31
```

**Comment by hideyukitone** [comment]
I am troubled with the same error.
Is there any solution?

**Comment by inamiy** [comment]
Same issue as #86, and fixed in #87.
